### ISSUE TITLE
Unable to use alongside github merge queues

### ISSUE THREAD

**Issue by GeorgePen93** [dorny/paths-filter#253]
We recently implemented github merge queues and this action no longer works due to how github creates a new branch for the merge queue.

We have tried to get around this issue by parsing the commit hash from the new github branch e.g. if the new merge queue branch is called `gh-readonly-queue/master/pr-2541-968707c39f80422b193c65cb50938119980604c5` we parse out the `968707c39f80422b193c65cb50938119980604c5` and pass this to path-filters.

This no longer causes an error but also does not work as intended as it now no longer will recognise changes in specific files when changes have been made to them in the commit that has been parsed from the branch name.

The only thing we have noticed is this error:

```
Searching for merge-base master...968707c39f80422b193c65cb50938119980604c5
  /usr/bin/git show-ref master
  /usr/bin/git cat-file -e 968707c39f80422b193c65cb50938119980604c5^{commit}
  fatal: Not a valid object name 968707c39f80[42](https://github.com/org/repo-name/actions/runs/11614801720/job/32343822442#step:6:44)2b193c65cb50938119980604c5^{commit}
  ```
  
  However the subsequent git fetch does succeed.

**Comment by TrueWill** [comment]
I'm not sure if my issue is the same bug:

```text
Get current git ref
  /usr/bin/git branch --show-current
  13399-ga-basic-pipeline
Changes will be detected between 757d8f7adf0b049dd679d3a232f191a8d29c0c60 and 13399-ga-basic-pipeline
Ensuring 757d8f7adf0b049dd679d3a232f191a8d29c0c60 is fetched from origin
  /usr/bin/git cat-file -e 757d8f7adf0b049dd679d3a232f191a8d29c0c60^{commit}
  fatal: Not a valid object name 757d8f7adf0b049dd679d3a232f191a8d29c0c60^{commit}
```

This is with `base: ${{ github.ref }}`.

**Comment by saschabratton** [comment]
@marcodicro-dp Are you able to pass the base and ref from the merge event, like as described in https://github.com/dorny/paths-filter/issues/183?

**Comment by marcodicro-dp** [comment]
Yes eventually I found that and have been using it as a workaround.

Will try removing the base and ref inputs now that #255 is merged, thank you!
